rabbitmq/include/userver/urabbitmq/component.hpp

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+/// @file userver/urabbitmq/component.hpp
+/// @brief @copybrief components::RabbitMQ
+
+#include <memory>
+
+#include <userver/components/loggable_component_base.hpp>
+
+#include <userver/utils/statistics/storage.hpp>
+
+USERVER_NAMESPACE_BEGIN
+
+namespace clients::dns {
+class Component;
+}
+
+namespace urabbitmq {
+class Client;
+}
+
+namespace components {
+
+// clang-format off
+/// @ingroup userver_components
+///
+/// @brief RabbitMQ (AMQP 0.9.1) client component
```

Comment:
I use `RabbitMQ` and `AMQP 0.9.1` almost interchangeably, because `AMQP 0.9.1` is a [core protocol](https://www.rabbitmq.com/protocols.html) of `RabbitMQ`, however the driver may partly work with other [implementations](https://en.wikipedia.org/wiki/Advanced_Message_Queuing_Protocol#Pre-1.0_AMQP_broker_implementations).

Things is, this version won't ever change, because the next one is `AMQP 1.0` which is completely different.